filedistribution/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-compress</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+        </dependency>
```

Comment:
I guess it was not used by commons-compress prior to 1.26.
Here it was only required in test scope, but since it was was used in constructor in I assume it would be needed in non-test scope, and I found nothing else pulling it in.

Reply:
https://mvnrepository.com/artifact/org.apache.commons/commons-compress/1.26.0
Has added commons-compress, commons-io and  commons-lang3 as dependency compare to 
https://mvnrepository.com/artifact/org.apache.commons/commons-compress/1.25.0